data/ipfs.go

```diff
@@ -303,13 +295,12 @@ func (i *IPFSHandle) Retrieve(ctx context.Context, path string, maxSize int64) (
 // The execution of this method might take a while (some minutes),
 // so the caller must handle properly the logic by using goroutines, channels or other
 // mechanisms in order to not block the whole program execution.
-func (i *IPFSHandle) PublishIPNSpath(ctx context.Context, path string,
-	keyalias string) (coreiface.IpnsEntry, error) {
-	root, err := i.AddAndPin(ctx, path)
-	if err != nil {
-		return nil, err
-	}
-	c, err := cid.Parse(root)
+func (i *IPFSHandle) PublishIPNSpath(
```

Comment:
I know some linters want to split the parameters this way, but we decided not to follow the approach. Instead we split the lines in the minimum number while trying not to surpass 111 chars and make them as symmetric as possible.

Reply:
good to know, i dropped that gofumptd commit